dependencies/test.py

```diff
@@ -0,0 +1,146 @@
+import os
+import pickle
+
+import requests
+from os.path import join, getsize
+import lxml.html
+
+from node import node
+
+
+class ReadFiles:
+    path=""
+
+    def __init__(self, path):
+        self.path = path
+
+    def getAllFiles(self):
+        list = []
```

Comment:
list is a python reserved keyword, please don't replace it with your local variable, rename it as flist.